Model/Api/Shipping.php

```diff
@@ -86,17 +86,21 @@ public function __construct(
     /**
      * @param array $addressData
      * @param null $shipping_option
+     * @param null $ship_to_store_option
      * @return ShippingDataInterface
      * @throws LocalizedException
      */
-    public function generateResult($addressData, $shipping_option)
+    public function generateResult($addressData, $shipping_option, $ship_to_store_option)
```

Comment:
Could we add comment about why we don't use the second and the third parameters?

Reply:
Added